I do magic/promises-callbacks-js/src/html/index.html

```diff
@@ -58,8 +36,8 @@
       <div class="image"></div>
       <div class="title">Los Mejores agentes de seguridad</div>
       <div class="searchbar">
-          <input type="text" class="searchbar__textfield" placeholder="Buscar por Ubicación" name="Location" onkeypress="validateEnter(event)">
-          <input type="image" src="images/search.png" class="searchbar__search-button" onclick="getAgents('https://api.myjson.com/bins/uptto')">
+          <input id="searchText" type="text" class="searchbar__textfield" placeholder="    Buscar por Ubicación">
```

Comment:
Why so many spaces at the beginning of the placeholder?

Reply:
Done